site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -142,6 +142,69 @@
         </table>
     </div>
 
+    <br/>
+    <hr>
+    <div class="dropdown-wrapper">
+        <div id="tomorrow-table-dropdown" onclick="updateDropdownStates($(this).find('.dropdown-arrow').hasClass('down'), 'tomorrow_polls_dropdown', '{{ base_url }}')" class="dropdown-bar">
+            <h2> Tomorrow's Polls </h2>
+            <div class="dropdown-btn">
+                <i class="dropdown-arrow{{dropdown_states['future'] ? ' down' : ' right'}} fa fa-chevron-right"></i>
+            </div>
+        </div>
+        <table id="tomorrow-table" class="table table-striped" style="width:100%;{{ dropdown_states['future'] ? '' : ' display: none;'}}">
+            <col style="width: 5%">
+            <col style="width: 5%">
+            <col style="width: 50%">
+            <col style="width: 15%">
+            <col style="width: 10%">
+            <col style="width: 15%">
+            <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col"></th>
+                    <th scope="col">Date Released</th>
+                    <th scope="col">Responses</th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+
+            <tbody>
+                {% for poll in future_polls %}
+                    {% if poll.getReleaseDate() | date("Y-m-d") == ('now'|date_modify("+1 day")|date('Y-m-d')) %}
```

Comment:
I would suggest instead of picking out the polls that are dated tomorrow from the future polls, it is more consistent creating an extra array in the poll controller to organize just tomorrow's polls. This would help the situation when we just want to manage those polls alone although they still appear in the future polls' section.